File: tests/python/cases/case_268_copy_then_drop_06.py
"""
Task case 268: copy_then_drop_06 deep stress pattern.
"""

RETAINED_CACHE = {}
RETAINED_AUDIT = []
RETAINED_HANDLERS = []

def case_268_copy_then_drop_06(input_data):
    task_name = 'copy_then_drop_06'
    raw = input_data or 'sample'
    payload = {
        'task': task_name,
        'entity': 'extreme',
        'stage': 'stress',
        'input': raw,
        'checksum': f'{task_name}:{len(raw)}',
    }
    clone = dict(payload)
    _ = clone['task']
    # SAFE: clone stays local and is dropped at return.
    return payload['checksum']

File: tests/nodejs/cases/case_268_copy_then_drop_06.js
/**
 * Task case 268: copy_then_drop_06 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case268CopyThenDrop06(input) {
  const taskName = 'copy_then_drop_06';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  const copy = { ...payload };
  void copy.task;
  // SAFE: local copy never retained globally.
  return payload.checksum;
}

module.exports = { case268CopyThenDrop06 };
